core/src/epicli/Dockerfile

```diff
@@ -5,6 +5,7 @@ ARG EPICLI_VERSION
 ARG USERNAME=epiuser
 ARG USER_UID=1000
 ARG USER_GID=$USER_UID
+ARG SHARED_DIR=/shared
```

Comment:
The the dockerized Epicli expects the /shared director as output. Check /epiphany/core/src/epicli/cli/helpers/Config.py.

My suggestion would be to replace"

`ENV DOCKER_CLI Yes` 

With:

ENV EPICLI_DOCKER_SHARED_DIR $SHARED_DIR

And change the logic of /epiphany/core/src/epicli/cli/helpers/Config.py to reflect this. When people build there own container with a changed SHARED_DIR argument it will always be reflected properly in the Epicli config when dockerized and avoid issues. Also it makes more sence then `ENV DOCKER_CLI Yes`